docs/getting-started/recurring-tasks.md

```diff
@@ -266,7 +266,7 @@ Examples of possible recurrence rules (mix and match as desired; these should be
 - `🔁 every 6 months on the 2nd Wednesday`
 - `🔁 every January on the 15th`
 - `🔁 every February on the last`
-- `🔁 every April and December on the 1st and 24th` (meaning every _April 1st_ and _December 24th_)
+- `🔁 every April and December on the 1st and 24th` (meaning every _April 1st_, _April 24th_, _December 1st_ and _December 24th_)
```

Comment:
I'd like to investigate this / read through #611 before making any additional comments.